README.md

```diff
@@ -91,6 +63,12 @@ This will start all the playground in watch mode.
 
 The project is built using [SolidJS](https://www.solidjs.com/). If you are new to SolidJS, we recommend checking out their [documentation](https://www.solidjs.com/docs) and [tutorials](https://www.solidjs.com/tutorial).
 
+### Suggested VSCode Extensions
+
+- Tailwinds
```

Comment:
I believe the name of the extension is slightly different, maybe?

But also, would it make sense to link out to the extensions on the VSCode marketplace?